tensorflow/python/kernel_tests/pooling_ops_test.py

```diff
@@ -843,7 +843,7 @@ def _testAvgPoolGradSamePadding3_1(self, data_format, use_gpu):
   def testShapeFunctionEdgeCases(self):
     # All shapes unknown.
     for pool_func in [tf.nn.max_pool, tf.nn.avg_pool]:
-      p = tf.nn.max_pool(tf.placeholder(tf.float32),
+      p = pool_func(tf.placeholder(tf.float32),
                          ksize=[1, 1, 1, 1], strides=[1, 1, 1, 1],
```

Comment:
I know it's annoying, but can you align these parameters?  Thanks for catching this.